Source/Linq/Mock.cs

```diff
@@ -57,7 +57,12 @@ public partial class Mock
 		/// <returns>The mocked object created.</returns>
 		public static T Of<T>() where T : class
 		{
-			return Mocks.CreateMockQuery<T>().First<T>();
+			//Disabled because of bad performance
+			//return Mocks.CreateMockQuery<T>().First<T>();
+			//and replaced by equivalent which is more than 10 times faster currently
```

Comment:
Hmm, IMHO using the word "bad" isn't quite fair. I totally agree that the current implementation has *worse* performance than your new one; but does that make it *bad*?

I would've chosen a slightly different wording, for example:

```csharp
// This method was originally implemented as follows:
//
//     return Mocks.CreateMockQuery<T>().First<T>();
//
// which involved a lot of (unnecessary) `IQueryable` query provider overhead and
// lambda compilation. What it really boils down to is this (much faster) code:
...
```